scripts/validationAndPreparation.sh

```diff
@@ -1,45 +1,64 @@
+#!/bin/bash
 
-FILES=`lftp -e "set ssl:key-file ~/.ssh/id_rsa; cls; bye" sftp://josojo:@trusted-setup.staging.gnosisdev.com`
+FILES=`lftp sftp://validationworker:@trusted-setup.staging.gnosisdev.com -e 'set sftp:connect-program "ssh -a -x -i /root/.ssh/id_rsa_worker";cls;bye'`
+
+#no contribution in folder challenges expected
+FILES=${FILES//$challenges/}
+NEWESTDATE=`cat /app/config/lastestContributionDate.txt`
 if [[ -z "${NEWESTDATE}" ]]; then
   NEWESTDATE=0
```

Comment:
Is this reachable given that we set it to 1 initially?

Reply:
Is no longer the case